docs/UserGuide.md

```diff
@@ -551,37 +569,38 @@ Acceptable values for `e/EMAIL`:
         - end with a domain label at least 2 characters long.
         - have each domain label start and end with alphanumeric characters.
         - have each domain label consist of alphanumeric characters, separated only by hyphens, if any.
-- Error message for invalid email: "Emails should contain two segments separated by an @ symbol."
+- Error message for invalid email: "Invalid command format!" followed by an instruction on the proper usage of the command you were executing.
```

Comment:
Not sure about this, seems like the current error message is slightly different?
```
Emails should be of the format local-part@domain and adhere to the following constraints:
1. The local-part should only contain alphanumeric characters and these special characters, excluding the parentheses, (+_.-). The local-part may not start or end with any special characters.
2. This is followed by a '@' and then a domain name. The domain name is made up of domain labels separated by periods.
The domain name must:
    - end with a domain label at least 2 characters long
    - have each domain label start and end with alphanumeric characters
    - have each domain label consist of alphanumeric characters, separated only by hyphens, if any.
```

Reply:
Resolved!